src/rigid_pendulum_poc/solver.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <iostream>
+#include <vector>
+
+#include <Kokkos_Core.hpp>
+
+namespace openturbine::rigid_pendulum {
+
+/// @brief Enumerate different types of solvers
+enum class SolverType {
```

Comment:
Do we need to define different solver types yet?  From a TDD perspective, we don't have a use for this functionality yet, so we should hold off on implementing it until we have a use case for it (as demonstrated by tests).